modules/test/poshi-runner/poshi-runner/src/test/resources/com/liferay/poshi/runner/dependencies/elements/PoshiScript.function

```diff
@@ -4,7 +4,7 @@ definition {
 	function dragAndDropPortletToObject {
 		WaitForSPARefresh();
 
-		selenium.waitForText("${locator1}", "${value1}");
+		selenium.waitForTextCaseInsensitive("${locator1}", "${value1}");
```

Comment:
@john-co Found something when trying to backport.

We cannot make changes to the poshi-runner module in master, it needs to be done through the subrepo. I suggest just not changing anything on the poshi-runner side.

Reply:
removed this change from poshi-runner modules